src/main/resources/specs/KsmService.yml

```diff
@@ -27,35 +27,8 @@ paths:
           schema:
             $ref: "#/definitions/GetAllAclsResponse"
 definitions:
-  KafkaPrincipalPb:
-    type: object
-    properties: 
-      principalType:
-        type: string
-      name:
-        type: string
-  ResourcePb:
-    type: object
-    properties: 
-      name:
-        type: string
-      kafkaResourceType:
-        type: string
-        enum:
-        - RESOURCE_TYPE_INVALID
-        - RESOURCE_TYPE_UNSET
-        - RESOURCE_TYPE_TOPIC
-        - RESOURCE_TYPE_GROUP
-        - RESOURCE_TYPE_CLUSTER
-        - RESOURCE_TYPE_TRANSACTIONALID
-        - RESOURCE_TYPE_DELEGATIONTOKEN
-  ResourceAndAclPb:
+  GetAllAclsRequest:
```

Comment:
can you revert that file please?
Overall I'll add this to a `.gitignore`. It's generated and giving too much problems